src/widgetastic/widget/base.py

```diff
@@ -950,45 +953,28 @@ def fill(self, values):
 
         Args:
             values: A dictionary of ``widget_name: value_to_fill``.
+            wait_widget: wait_for value. If wait > 0 widget.wait_displayed(wait) is called for every
```

Comment:
artifact from first version. forgot to remove this. it will be removed in a bit.

Reply:
bump, still extra arg.